docs/formatting.md

```diff
@@ -35,7 +35,7 @@ Assuming you installed with the typical layout:
 
 ### Re-format specific file(s)
 
-`bazel run tools:format some/file.md other/file.json`
+`bazel run format some/file.md other/file.json`
```

Comment:
switch this one to `bazel run //:format ...` as well as the one above for consistency?